### ISSUE TITLE
Add missing translations for ARIA attributes & image alt text

### ISSUE THREAD

**Issue by clhenrick** [clhenrick/am-i-rent-stabilized#126]
PR #125 made use of `aria-label` and `aria-roledescription` in [multiple places in `main.hbs`](https://github.com/clhenrick/am-i-rent-stabilized/blob/f74979c2a4a395bc1fd3c9fcf86c3a66d04e9eff/app/src/hbs_templates/main.hbs#L15-L253) to make the homepage more accessible. However the values of these attributes are currently hard coded strings. They should be translated to support languages other than English and make use of the Handlebars templating for rendering text.

There is also [image alt text](https://github.com/clhenrick/am-i-rent-stabilized/blob/f74979c2a4a395bc1fd3c9fcf86c3a66d04e9eff/app/src/hbs_templates/main.hbs#L194) for the rent history modal overlay that needs to be translated.
